Documentation/install/system_requirements.rst

```diff
@@ -48,6 +48,15 @@ iproute2                 >= 5.0.0 [#iproute2_foot]_ yes
 .. [#iproute2_foot] Requires support for eBPF templating as documented
    :ref:`below <iproute2_requirements>`.
 
+.. hint:: 
+
+    ``cilium kernel-check`` can be invoked  as a Kubernetes Job  
+    in order to check whether the machine is fulfilling  eBPF capabilities: 
```

Comment:
Or maybe `provides` instead? I think this would be a bit more natural:

```suggestion
    ``cilium kernel-check`` can be invoked as a Kubernetes Job  
    in order to check whether the machine provides the relevant eBPF capabilities: 
```